exams/management/commands/populate_edx_exam_coupons.py

```diff
@@ -34,7 +34,7 @@ def handle(self, *args, **kwargs):  # pylint: disable=unused-argument,too-many-l
         csvfile = kwargs.get('csvfile')
         reader = csv.DictReader(csvfile.read().splitlines())
         catalog_query = next(reader)['Catalog Query']
-        course_number = re.search(r"key:\(([A-Za-z0-9.]+)", catalog_query).group(1)
+        course_number = re.search(r"\+([A-Za-z0-9.]+)PEx", catalog_query).group(1)
```

Comment:
Do we care about the old pattern anymore?

Reply:
I don't think we need the old pattern. It looks like the exam code is going to be just like edx course key, but with PEx after the number, like this `MITx+13.456PEx+3T2020`